app/models/bookmark.rb

```diff
@@ -110,7 +110,7 @@ def invalidate_bookmark_count
   # We index the bookmark count, so if it should change, update the pseud
   def update_pseud_index
     return unless $rollout.active?(:start_new_indexing)
-    return unless destroyed? || saved_change_to_id? || saved_change_to_private?
+    return unless destroyed? || saved_change_to_id? || saved_change_to_private? || saved_change_to_hidden_by_admin?
```

Comment:
This covers fields similar to the work counterpart:

https://github.com/otwcode/otwarchive/blob/abab68178842662391095acdc206f1b114ab5649/app/models/work.rb#L271-L275

so I don't think anything else is missing.